test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_horovod.py

```diff
@@ -30,7 +30,7 @@
 @pytest.mark.model("mnist")
 @pytest.mark.multinode(2)
 def test_distributed_training_horovod(
-    ecr_image, sagemaker_regions, instance_type, tmpdir, framework_version
+    ecr_image, sagemaker_regions, instance_type, tmpdir, framework_version, tf212_and_below_only
```

Comment:
I don't think `tf212_and_below_only` exists as a fixture for use by SM tests. The `conftest.py` for these tests doesn't have this fixture.

Reply:
created `sm_below_tf213_only` fixture to support the SM work that will be in the next PR.